### DISCUSSION TITLE
Testing changes from issue #1028 (conditionals with ../)

### DISCUSSION THREAD

**Discussion by martiniert** [handlebars-lang/handlebars.js#1800] [Q&A]
Currently I'm still using old handlebars version 3.0.0 because I've got a huge project with over 700 instances of "../".
In handlebars 4.0.0, the issue https://github.com/handlebars-lang/handlebars.js/issues/1028 was fixed, therefore all instances of ../ within conditionals  need to be retested.

Is there a another way to test the behaviour of the conditional context switches then just clicking through the whole project by hand?
Is is possible to give a console warning where the ../ now behaves differently than before?

**Comment by jaylinski** [comment]
I'm afraid you have to write tests for your application in order to find out about broken templates after upgrading to v4.

Another possibility would be to use a custom version of handlebars.js v4, where you would insert `console.warning()` in the two places touched by this commit: https://github.com/handlebars-lang/handlebars.js/commit/279e038ba7ff7e5966a60e36d326e2d4c310f0c6
